src/main/java/com/cicd/helloworld/Main.java

```diff
@@ -2,6 +2,6 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world\n");
+        System.out.println("Hello world my friends from UMSS!!!!\n");

```

Comment:
Please, It's not necessary to repeat the expression character (!), just one is enough.